python/ray/rllib/evaluation/tf_policy_graph.py

```diff
@@ -202,7 +207,12 @@ def export_model(self, export_dir):
 
     @override(PolicyGraph)
     def export_checkpoint(self, export_dir, filename_prefix="model"):
-        """Export tensorflow checkpoint to export_dir."""
+        """Export tensorflow checkpoint to export_dir.
+           Existing dir will be overwritten.
+        """
+        if os.path.exists(export_dir):
+            shutil.rmtree(export_dir)
```

Comment:
This seems like it could surprise users and perhaps delete their home directory. Should we instead raise an error if the directory already exists?

Reply:
Fixed. TensorFlow will report error while exporting model to an existing directory and auto-overwrite while exporting checkpoint.